modules/tinymce/src/core/test/ts/browser/bookmark/BookmarksTest.ts

```diff
@@ -223,4 +223,50 @@ describe('browser.tinymce.core.bookmark.BookmarksTest', () => {
     assertApproxRawContent(editor, '<p>abc</p>');
     TinyAssertions.assertSelection(editor, [ 0, 0 ], 1, [ 0, 0 ], 2);
   }));
+
+  context('Get bookmark should work if selection is on a comment', () => {
```

Comment:
This all seems very outdated now... the labels and function names need changing because we aren't putting the selection in the comment anymore.

Reply:
ok, I fixed the texts in `context` and in the `it`s